### PR TITLE
Saltstack Plugin update

### PR DESCRIPTION
**Pull request by j-warchol**
- documentation (`README.md`)
- unittests
- system test (it was developed in parallel with adding new features, so it evolved over time)
- minion authorization using salt wheels
- minor fixes, refactoring, bugfixing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,266 @@
...
+                eauth: configuration here
+
+
+### Minimum working example ###
+
+The following is a basic working example:
+
+        tosca_definitions_version: cloudify_dsl_1_0
+        imports:
+            - http://www.getcloudify.org/spec/cloudify/3.1rc1/types.yaml
```
Comment (idanmo) [inline README.md]: Are you going to change the 3.1rc1 references to 3.1?

Comment (j-warchol) [inline README.md]: Ah, I had overlooked that, sorry.  Updated in latest commit.

**Comment by barakm** [comment]
Hi @j-warchol 
This PR was pretty much forgotten. What is its status?

**Comment by j-warchol** [comment]
Well, it's been half a year since I've been working on cloudify, but as far
as I can see this pull request represents the finished version of SaltStack
plugin that we developed back then.

On Mon, Jul 6, 2015 at 1:20 PM, barakm <notifications@github.com> wrote:

> Hi @j-warchol <https://github.com/j-warchol>
> This PR was pretty much forgotten. What is its status?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-saltstack-plugin/pull/2#issuecomment-118985638>
> .
>



-- 
*Jan Warchoł*
*Software** Engineer*


-----------------------------------------
M: +48 509 078 203
E: jan.warchol@codilime.com
-----------------------------------------

CodiLime Sp. z o.o. - Ltd. company with its registered office in Poland,
01-167 Warsaw, ul. Zawiszy 14/97. Registered by The District Court for the
Capital City of Warsaw, XII Commercial Department of the National Court
Register. Entered into National Court Register under No. KRS 0000388871.
Tax identification number (NIP) 5272657478. Statistical number (REGON)
142974628.

-----------------------------------------

The information in this email is confidential and may be legally
privileged, it may contain information that is confidential in CodiLime Sp.
z o.o. It is intended solely for the addressee. Any access to this email by
third parties is unauthorized. If you are not the intended recipient of
this message, any disclosure, copying, distribution or any action
undertaken or neglected in reliance thereon is prohibited and may result in
your liability for damages.

**Comment by barakm** [comment]
This Pull Request is very old, far too old to be updated. I am closing it, and we will re-visit the subject in the future.

**Pull request closed without merging**